src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -1488,7 +1488,7 @@ export default defineComponent({
 
       const format = screenshotFormat.value
       const mimeType = `image/${format === 'jpg' ? 'jpeg' : format}`
-      const imageQuality = format === 'jpg' ? screenshotQuality.value / 100 : 1
+      const imageQuality = format !== 'png' ? screenshotQuality.value / 100 : 1
```

Comment:
Just throw the value into it without checking format?
For png it doesn't change anything

Reply:
So replace the ternary with a code comment? We need something there.

Reply:
Just throw `screenshotQuality.value / 100` in and maybe with a comment about png format doesn't care about it

Reply:
Done